### PR TITLE
Update french translations

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
Continue work on french translations started here #22646

### CODE REVIEW

**Comment by LonwoLonwo** [comment]
Hello @Jean-BaptisteC 

Thanks for your contibution.
We will take a look at it.

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_fr.properties [hunk 1]
```diff
@@ -137,4 +137,33 @@ confirm.driver_download.title = T\u00E9l\u00E9charger le pilote de base de donn\
...
+command.org.jkiss.dbeaver.core.connection.readonly.name=Lecture seule
+command.org.jkiss.dbeaver.core.connection.synchronize.name=Synchroniser
+command.org.jkiss.dbeaver.core.connection.synchronize.description=Synchronisez les sources de donn\u00E9es locales et distantes
+command.org.jkiss.dbeaver.core.qm.delete.name=Supprimer les requ\u00EAtes du journal
+command.org.jkiss.dbeaver.core.qm.delete.description=Supprimer les requ\u00EAtes s\u00E9lectionn\u00E9s du journal
+page.org.jkiss.dbeaver.preferences.editors.name=Editeurs
+page.org.jkiss.dbeaver.preferences.accessibility.name=Accessibilit\u00E9
+wizard.org.jkiss.dbeaver.core.wizard.export.workspace.name=Pr\u00E9f\u00E9rences
+wizard.org.jkiss.dbeaver.core.wizard.import.workspace.name=Pr\u00E9f\u00E9rences
+confirm.disable_network_handler.message=Etes vous sur de d\u00E9sactiver la configuration {0} ?
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_fr.properties:161]:
```suggestion
confirm.disable_network_handler.message=Etes vous sur de d\u00E9sactiver la configuration {0}?
```

File: plugins/org.jkiss.dbeaver.data.office.ui/OSGI-INF/l10n/bundle_fr.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+category.org.jkiss.dbeaver.ext.data.office.description=Commandes Office
+category.org.jkiss.dbeaver.ext.data.office.name=Office
+command.org.jkiss.dbeaver.ext.data.office.results.openSpreadsheet.name=Ouvrir les r\u00e9sultats dans Excel
+command.org.jkiss.dbeaver.ext.data.office.results.openSpreadsheet.description=Ouvrir les r\u00e9sultats dans Excel/Cal
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.data.office.ui/OSGI-INF/l10n/bundle_fr.properties]:
```suggestion
command.org.jkiss.dbeaver.ext.data.office.results.openSpreadsheet.description=Ouvrir les r\u00e9sultats dans Excel/Calс
```

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.data.office.ui/OSGI-INF/l10n/bundle_fr.properties]: This is the program name.

File: plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_fr.properties [hunk 3]
```diff
@@ -8,12 +8,12 @@ dataTransfer.processor.csv.description                          = Exporter en fi
 dataTransfer.processor.csv.name                                 = CSV
 dataTransfer.processor.csv.property.delimiter.description       = S\u00E9parateur de colonnes
 dataTransfer.processor.csv.property.delimiter.name              = S\u00E9parateur
-dataTransfer.processor.csv.property.escape.description          = Mod\u00E8le d'\u00E9chappement de caract\u00E8res sp\u00E9ciaux (entourr\u00E9s de guillemets ou \u00E9chapp\u00E9s par le caract\u00E8re '\' )
-dataTransfer.processor.csv.property.escape.name                 = Echappement de caract\u00E8res
+dataTransfer.processor.csv.property.escape.description=Mod\u00E8le d'\u00E9chappement de caract\u00E8res sp\u00E9ciaux (entourr\u00E9s de guillemets ou \u00E9chapp\u00E9s par le caract\u00E8re '' )
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_fr.properties]:
```suggestion
dataTransfer.processor.csv.property.escape.description=Mod\u00E8le d'\u00E9chappement de caract\u00E8res sp\u00E9ciaux (entourr\u00E9s de guillemets ou \u00E9chapp\u00E9s par le caract\u00E8re '\\' )
```

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_fr.properties]: Return the correct value from the original English message.

File: plugins/org.jkiss.dbeaver.debug.core/OSGI-INF/l10n/bundle_fr.properties [hunk 4]
```diff
@@ -0,0 +1,4 @@
+launchConfigurationType.dbeaver.name=Dbeaver
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.debug.core/OSGI-INF/l10n/bundle_fr.properties]:
```suggestion
```

File: plugins/org.jkiss.dbeaver.debug.core/OSGI-INF/l10n/bundle_fr.properties [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+launchConfigurationType.dbeaver.delegateName=Dbeaver
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.debug.core/OSGI-INF/l10n/bundle_fr.properties]: ### 
```suggestion
```

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.debug.core/OSGI-INF/l10n/bundle_fr.properties]: No need to translate our name.

File: plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_fr.properties [hunk 6]
```diff
@@ -0,0 +1,37 @@
+editor.erd.name=Diagramme ER
+editor.erd.description=Diagramme entit\u00e9-relation
+menu.create.new.diagram=Cr\u00e9er un nouveau diagrame ERD
+pref.page.name.erd.editor=Editeur de diagramme
+page.org.jkiss.dbeaver.preferences.erd.general.name=Param\u00e8tres ERD
+wizard.erd.name=Diagramme ERD
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_fr.properties]:
```suggestion
wizard.erd.name=Diagramme ER
```

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_fr.properties]: ERD = Entity Relationship Diagram

File: plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_fr.properties [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+wizard.erd.description=Diagramme entit\u00e9-relation (ERD)
+command.org.jkiss.dbeaver.erd.diagram.view.name=Voir le diagramme
+command.org.jkiss.dbeaver.erd.diagram.view.description=voir le diagramme entit\u00e9-relation
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_fr.properties]:
```suggestion
command.org.jkiss.dbeaver.erd.diagram.view.description=Voir le diagramme entit\u00e9-relation
```

File: plugins/org.jkiss.dbeaver.ext.altibase/OSGI-INF/l10n/bundle_fr.properties [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+tree.roles.node.name=R\u00f4les
+tree.schemas.node.name=Sch\u00e9mas
+tree.security.node.name=S\u00e9curit\u00e9
+tree.sequence.node.name=S\u00e9quence
+tree.sequences.node.name=S\u00e9quences
+tree.sequences.node.tip=S\u00e9quences
+tree.storage.node.name=Stockage
+tree.sub_table.node.name=Sous-table
+tree.synonym.node.name=Synonyme
+tree.synonyms.node.name=Synonyme
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.ext.altibase/OSGI-INF/l10n/bundle_fr.properties]: Synonymes?

File: plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_fr.properties (hunk 1 above)
Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_fr.properties:161]: In french, we have space between text and punctuation :)

File: plugins/org.jkiss.dbeaver.data.office.ui/OSGI-INF/l10n/bundle_fr.properties (hunk 2 above)
Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.data.office.ui/OSGI-INF/l10n/bundle_fr.properties]: Done

File: plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_fr.properties (hunk 3 above)
Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_fr.properties]: Reverted

File: plugins/org.jkiss.dbeaver.debug.core/OSGI-INF/l10n/bundle_fr.properties (hunk 4 above)
Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.debug.core/OSGI-INF/l10n/bundle_fr.properties]: Done

File: plugins/org.jkiss.dbeaver.debug.core/OSGI-INF/l10n/bundle_fr.properties (hunk 5 above)
Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.debug.core/OSGI-INF/l10n/bundle_fr.properties]: Done

File: plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_fr.properties (hunk 6 above)
Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_fr.properties]: Update to `Diagramme entité-association`

File: plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_fr.properties (hunk 7 above)
Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_fr.properties]: Done

File: plugins/org.jkiss.dbeaver.ext.altibase/OSGI-INF/l10n/bundle_fr.properties (hunk 8 above)
Comment (Jean-BaptisteC) [reply plugins/org.jkiss.dbeaver.ext.altibase/OSGI-INF/l10n/bundle_fr.properties]: Nice catch

**Review by LonwoLonwo — APPROVED** [review]

**Comment by LonwoLonwo** [comment]
Now we are waiting for more one review and QA-Team.

**Review by E1izabeth — APPROVED** [review]

**Comment by MashaKorax** [comment]
verified

**Pull request merged**